GrandCentralBoard/Widgets/GitHub/GitHubDataProvider.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  GitHubDataProvider.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 27.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import GrandCentralBoardCore
+import Moya
+import RxSwift
+import Decodable
+
+
+enum GitHubTarget: TargetType {
+    case Repositories
+    case PullRequests(repositoryFullName: String)
+
+    var baseURL: NSURL { return NSURL(string: "https://api.github.com")! }
+
+    var path: String {
+        switch self {
+        case .Repositories: return "/user/repos"
+        case .PullRequests(let repositoryFullName): return "/repos/\(repositoryFullName)/pulls"
+        }
+    }
+
+    var method: Moya.Method { return .GET }
+    var parameters: [String : AnyObject]? { return nil }
+    var sampleData: NSData { return NSData() }
+}
+
+
+protocol GitHubDataProviding {
+    func requestRepositories() -> Observable<[Repository]>
+    func requestRepositoriesWithPRs() -> Observable<[Repository]>
```

Comment:
Wouldn't it be enough to use these names:

```
    func repositories() -> Observable<[Repository]>
    func repositoriesWithPRs() -> Observable<[Repository]>
```

Do we care that there's some request performed under the hood?